src/Components/Components/src/ICascadingValueSupplierFactory.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Components;
+
+internal interface ICascadingValueSupplierFactory
+{
+    bool TryGetValueSupplier(object attribute, Type parameterType, string parameterName, [NotNullWhen(true)] out ICascadingValueSupplier? result);
```

Comment:
Can you comment on why `attribute` is of type `object` here and not `Attribute`?

Reply:
I've since changed this 🙂